modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspacePlugin.java

```diff
@@ -84,6 +85,8 @@ public void doCall(Project project) {
 					Plugin<Project> plugin = null;
 
 					if (project.getParent() == null) {
+						_applyPlugins(project);
```

Comment:
See my comment [here](https://github.com/drewbrokke/liferay-portal/pull/1442#discussion_r1211700547).

Reply:
ok I will move it to first line.